src/Analyzer.TemplateProcessor/ArmTemplateProcessor.cs

```diff
@@ -403,7 +404,8 @@ private void ProcessTemplateResourceLanguageExpressions(TemplateResource templat
             {
                 // Do not throw if there was an issue with evaluating language expressions
 
-                logger?.LogWarning(ex, "An exception occurred while evaluating the properties of a resource. Properties: {properties}", templateResource.Properties.Value);
+                logger?.LogWarning(ex, "An exception occurred while evaluating the properties of a resource");
```

Comment:
Can you include the path to the property in the warning?  It will make investigation easier without needing to run in verbose.

Reply:
We don't have the path to the particular property that failed parsing, only to the properties object